lib/dogma/formatter.ex

```diff
@@ -1,56 +1,11 @@
 defmodule Dogma.Formatter do
```

Comment:
Can we call these reporters or something now? They don't just format data any more.